### ISSUE TITLE
Fix tooltips

### ISSUE THREAD

**Issue by andrechalom** [lageIBUSP/Rsampling-shiny#6]
The tooltip from shinyBS is appearing on the "div" element outside the table, which is stretched to the page width. This causes the popup to appear displaced from the table itself:

![shiny](https://cloud.githubusercontent.com/assets/1994997/8397476/8aa25e5a-1da2-11e5-9830-d4885646c0ba.png)

**Comment by andrechalom** [comment]
Also, the tooltip for textInput is not appearing. The following code does not generate tooltips, so I'm switching to helpText:

``` R
conditionalPanel("input.stat == 'custom'",
  textInput("customcols", "Columns", "1")
),
bsTooltip("customcols", "Which columns of the data set should be randomized? This input
               will be parsed as R code, so 1:3 or c(1,4,5) are valid values", "bottom"),
```

**Comment by piklprado** [comment]
A feedback from our first testers: many of the students in the field course found the tooltips more annoying than useful, specially because they keep hiding parts of the dialogs, options and so on. Maybe better to move the information in tooltips to help texts in the pages?

**Comment by piklprado** [comment]
I suggest to assign this issue to @adalardo , anyway.

**Comment by andrechalom** [comment]
Added @piklprado @adalardo and @ayanamartins as collaborators in this repository, so you can commit changes directly to the repository, merge pull request, label issues, close issues, and be assigned for tasks.

On the topic: I have mixed feelings about the tooltips. I think the tooltips can be a great way to "hide" small help texts, but I feel that our help messages _need_ to be large, and so the tooltips may become an obstruction. Compare with the tooltips present in github: the small pencil near the comments has the tooltip "Edit comment", with only TWO WORDS.

If we decide to remove tooltips altogether, we can drop the dependence on shinyBS.

**Comment by adalardo** [comment]
I like tooltips for short messages like two up to four letters. For  more words we should use help messages!

**Comment by piklprado** [comment]
Agree with tooltips with few words.

**Comment by andrechalom** [comment]
Working on a version without toolips

**Comment by andrechalom** [comment]
Implemented on #34, please send feedback.
